demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -70,9 +70,14 @@ def run_validations(self) -> int:
                 validation_results: List[ValidationResult] = validator.is_valid(filtered_content_objects_for_validator)  # type: ignore
                 if self.allow_autofix and validator.is_auto_fixable:
                     for validation_result in validation_results:
-                        self.validation_results.append_fix_results(
-                            validator.fix(validation_result.content_object)  # type: ignore
-                        )
+                        try:
+                            self.validation_results.append_fix_results(
+                                validator.fix(validation_result.content_object)  # type: ignore
```

Comment:
```suggestion
                                validator.fix(validation_result.content_object)  # type: ignore[error-code]
```

even though i see mypy ignored it, please put the error-code / reason we should ignore mypy